features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapdConfigBean.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
```

Comment:
This looks good. We need this bean to encapsulate the config on the Minion side.